cogs/utility.py

```diff
@@ -23,6 +25,8 @@ def __init__(self, bot: discord.Bot):
             required=False)
     async def sonagen(self, ctx: discord.ApplicationContext, species: str, sex: str, sonatype: str):
         """ Generate a random sona """
+        if not self.ai_enabled:
+            return await ctx.respond("AI functions are disabled due to missing (or invalid) API key, please contact the bot owner to fix this.")
```

Comment:
Make the response ephemeral, just say "AI functions are currently disabled" without the last part and log it instead (look at how I log stuff elsewhere in the bot)